ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -0,0 +1,92 @@
+try:
+    from tqdm import tqdm
+except ImportError:
+    raise RuntimeError("This contrib module requires tqdm to be installed")
+
+from ignite.engine import Events
+
+
+class ProgressBar:
+    """
+    TQDM progress bar handler to log training progress and computed metrics
+
+    Args:
+        engine (ignite.Engine): an engine object
+        loader (iterable or DataLoader): data loader object
+        output_transform: transform a function that transforms engine.state.output
+                into a dictionary of format {metric_name: metric_value}
+        mode (str): 'iteration' or 'epoch' (default=epoch)
+        log_interval (int or None): interval of which the metrics information is displayed.
+                            If set to None, only the progress bar is shown and not
+                            the metrics. (default=1)
+
+    Example:
+        (...)
+        pbar = ProgressBar(trainer, train_loader, output_transform=lambda x: {'loss': x})
+        trainer.add_event_handler(Events.ITERATION_COMPLETED, pbar)
```

Comment:
What do you think to change a little the API as 
```
pbar = ProgressBar(num_iterations, metric_names)
pbar.attach(trainer, mode='epoch', log_interval=1)
# or
pbar.attach(trainer, mode='iteration', log_interval=10)
```

Reply:
looks good to me!

Reply:
but I wouldn't pass `num_iterations` if we are attaching it to the trainer anyway